### ISSUE TITLE
Bundle Audit giving error

### ISSUE THREAD

**Issue by sashank** [rubysec/bundler-audit#208]
hi am getting below error please help

```
bundle audit check
Traceback (most recent call last):
	19: from /usr/local/bin/bundler-audit:23:in `<main>'
	18: from /usr/local/bin/bundler-audit:23:in `load'
	17: from /var/lib/gems/2.5.0/gems/bundler-audit-0.6.1/bin/bundler-audit:3:in `<top (required)>'
	16: from /var/lib/gems/2.5.0/gems/bundler-audit-0.6.1/bin/bundler-audit:3:in `load'
	15: from /var/lib/gems/2.5.0/gems/bundler-audit-0.6.1/bin/bundle-audit:10:in `<top (required)>'
	14: from /var/lib/gems/2.5.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
	13: from /var/lib/gems/2.5.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
	12: from /var/lib/gems/2.5.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
	11: from /var/lib/gems/2.5.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
	10: from /var/lib/gems/2.5.0/gems/bundler-audit-0.6.1/lib/bundler/audit/cli.rb:41:in `check'
	 9: from /var/lib/gems/2.5.0/gems/bundler-audit-0.6.1/lib/bundler/audit/cli.rb:41:in `new'
	 8: from /var/lib/gems/2.5.0/gems/bundler-audit-0.6.1/lib/bundler/audit/scanner.rb:45:in `initialize'
	 7: from /var/lib/gems/2.5.0/gems/bundler-audit-0.6.1/lib/bundler/audit/scanner.rb:45:in `new'
	 6: from /var/lib/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/lockfile_parser.rb:68:in `initialize'
	 5: from /var/lib/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/lockfile_parser.rb:68:in `new'
	 4: from /var/lib/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/source/rubygems.rb:24:in `initialize'
	 3: from /var/lib/gems/2.5.0/gems/bundler-2.0.1/lib/bundler/source/rubygems.rb:462:in `cache_path'
	 2: from /var/lib/gems/2.5.0/gems/bundler-2.0.1/lib/bundler.rb:251:in `app_cache'
	 1: from /var/lib/gems/2.5.0/gems/bundler-2.0.1/lib/bundler.rb:233:in `root'
/var/lib/gems/2.5.0/gems/bundler-2.0.1/lib/bundler.rb:237:in `rescue in root': Could not locate Gemfile or .bundle/ directory (Bundler::GemfileNotFound)
```

**Comment by tbuehlmann** [comment]
> Could not locate Gemfile or .bundle/ directory (Bundler::GemfileNotFound)

Are you running the command in a directory that has a Gemfile?

**Comment by postmodern** [comment]
Fixed in 021f85f1a7daaad5e46a052e3788c552379387b6. If the `Gemfile.lock` is missing an exception will be raised, rescued in `CLI#check`, and an error message printed.
